test/language/block-scope/syntax/leave/try-block-let-declaration-only-shadows-outer-parameter-value-1.js

```diff
@@ -0,0 +1,16 @@
+// Copyright (C) Copyright 2011 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 13.1
+description: >
+    try block let declaration only shadows outer parameter value 1
+---*/
+(function(x) {
```

Comment:
For these tests, do you think it matters that the outermost `x` declaration is made via an arguments list (`var` semantics) and not a `let` statement? I guess they're identical in this context, but I'm wondering if using arguments here could hide weird implementation bugs.